src/main/java/uk/gov/hmcts/reform/et/syaapi/config/OpenAPIConfiguration.java

```diff
@@ -7,17 +7,20 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import static uk.gov.hmcts.reform.et.syaapi.constants.EtSyaConstants.MIT_LICENSE;
+import static uk.gov.hmcts.reform.et.syaapi.constants.EtSyaConstants.REMOTE_REPO;
+
 @Configuration
 public class OpenAPIConfiguration {
 
     @Bean
     public OpenAPI openAPI() {
         return new OpenAPI()
             .info(new Info().title("et-sya-api")
-                     .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
+                      .license(new License().name("MIT").url(MIT_LICENSE)))
```

Comment:
Why make these constants in another class? Are you expecting them to be used elsewhere?

Reply:
Won't use anywhere else so reverted